maps/mongodb_connector.py

```diff
@@ -17,42 +16,59 @@
 # Delete current content of collection
 collection.delete_many({})
 
-# Load embedding data into mongodb
+# Load embedding data into MongoDB
 mongo_so = pd.read_parquet(Path.cwd() / 'data' / 'mongo-so.parquet')
+
+# Initialize SentenceTransformer model
 model = SentenceTransformer('all-MiniLM-L6-v2')
-title_embeds = model.encode(mongo_so['title'])
+
+# Encode titles into embeddings
+title_embeds = model.encode(mongo_so['title'].tolist())
+
+# Assign embeddings to DataFrame
 mso_te = mongo_so.assign(title_embedding=list(title_embeds))
 
-data = list(r._asdict() for r in mso_te.itertuples())
+# Convert DataFrame to list of dictionaries for MongoDB insertion
+data = mso_te.to_dict(orient='records')
 for d in data:
     del d['Index']
     d['title_embedding'] = d['title_embedding'].tolist()
-data[0]
+
+# Insert data into MongoDB collection
 collection.insert_many(data)
 
-# Read a mongodb collection with embeddings in it and map it:
-project = AtlasProject(
-    name='MongoDB Stack Overflow Questions',
-    unique_id_field='mongo_id',
-    reset_project_if_exists=True,
+# Read MongoDB collection with embeddings and map it using AtlasDataset
+dataset = AtlasDataset(
+    "MongoDB_StackOverflow_Questions",
+    unique_id_field="mongo_id",
+    reset_dataset_if_exists=True,
```

Comment:
this parameter doesn't exist anymore, you should spin up mongodb locally with a collection to test this